src/corelibs/U2View/src/ov_msa/ScrollController.cpp

```diff
@@ -346,9 +347,29 @@ GScrollBar* ScrollController::getVerticalScrollBar() const {
     return vScrollBar;
 }
 
-void ScrollController::sl_zoomScrollBars() {
-    zoomHorizontalScrollBarPrivate();
-    zoomVerticalScrollBarPrivate();
+void ScrollController::updateScrollBarsOnFontOrZoomChange() {
+    CHECK(!maEditor->isAlignmentEmpty(), );
+    SignalBlocker signalBlocker(hScrollBar);
```

Comment:
We use `QSignalBlocker` now, usually, `SignalBlocker` is a bit deprecated, it was used when `QSignalBlocker` wasn't implemented yet.

Reply:
Fixed